Source/WebCore/dom/Position.cpp

```diff
@@ -489,7 +491,8 @@ bool Position::atLastEditingPositionForNode() const
     if (isNull())
         return true;
     // FIXME: Position after anchor shouldn't be considered as at the first editing position for node since that position resides outside of the node.
-    return m_anchorType == PositionIsAfterAnchor || m_anchorType == PositionIsAfterChildren || m_offset >= static_cast<unsigned>(lastOffsetForEditing(*protectedDeprecatedNode()));
+    auto anchorNode = protectedDeprecatedNode();
+    return m_anchorType == PositionIsAfterAnchor || m_anchorType == PositionIsAfterChildren || m_offset >= static_cast<unsigned>(lastOffsetForEditing(*anchorNode));
```

Comment:
Same comment, I believe the code was safe before and more concise. I personally prefer the old code.

Reply:
Reverted.